### PR TITLE
Update getting-started.asciidoc

### PR DESCRIPTION
**Pull request by dhimanrahul**
exact command for enabling winlogbeat.ps1 execution is not the one mentioned in the article  and also steps after running the command will be different.
below are the changes, i have recommended.
PowerShell.exe -ExecutionPolicy UnRestricted -File "C:\Program Files\winlogbeat\install-service-winlogbeat.ps1"
Than you would require to press "R".
and than you have to manually start the service from services.msc.

## Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have made corresponding change to the default configuration files
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added an entry in `CHANGELOG.next.asciidoc` or `CHANGELOG-developer.next.asciidoc`.

## Author's Checklist

- [ ]

## Related issues

-

### CODE REVIEW

File: winlogbeat/docs/getting-started.asciidoc [hunk 1]
```diff
@@ -55,7 +55,9 @@ Stopped  winlogbeat         winlogbeat
 
 NOTE: If script execution is disabled on your system, you need to set the
 execution policy for the current session to allow the script to run. For example:
-`PowerShell.exe -ExecutionPolicy UnRestricted -File .\install-service-winlogbeat.ps1`.
+PowerShell.exe -ExecutionPolicy UnRestricted -File "C:\Program Files\winlogbeat\install-service-winlogbeat.ps1"
+Than you would require to press "R".
```
Comment (ShourieG) [inline winlogbeat/docs/getting-started.asciidoc:59]: Spelling mistake Than -> Then

File: winlogbeat/docs/getting-started.asciidoc [hunk 2]
```diff
@@ -55,7 +55,9 @@ Stopped  winlogbeat         winlogbeat
...
+and than you have to manually start the service from services.msc.
```
Comment (ShourieG) [inline winlogbeat/docs/getting-started.asciidoc:60]: same here.  

Also I think the part where you describe the execution of the script can be better organised in a step-wise fashion.

**Comment by ShourieG** [comment]
@dhimanrahul Please sign the [Contributor Agreement](https://www.elastic.co/contributor-agreement) , in order to get your PR's approved and merged.

**Comment by epixa** [comment]
@dhimanrahul Thanks for your contribution, but we cannot proceed without the CLA being signed. If you'd like to proceed, please feel free to sign the CLA and open a new PR, and we can get this reviewed.

**Pull request closed without merging**